collections/date-time/format-relative-time.md

```diff
@@ -0,0 +1,15 @@
+```javascript
+const rtf = (value, unit) => new Intl.RelativeTimeFormat('en').format(value, unit);
```

Comment:
It would be great if the local could be passed as an optional parameter.
E.g.
`const rtf = (value, unit, locale = 'en') => new Intl.RelativeTimeFormat(locale).format(value, unit);`